packages/xchain-cosmos/src/cosmos/sdk-client.ts

```diff
@@ -224,29 +225,73 @@ export class CosmosSDKClient {
     return this.signAndBroadcast(unsignedStdTx, privkey, AccAddress.fromBech32(from))
   }
 
-  async sign(unsignedStdTx: StdTx, privkey: PrivKey, signer: AccAddress): Promise<StdTx> {
+  async signAndBroadcast(unsignedStdTx: StdTx, privkey: PrivKey, signer: AccAddress): Promise<BroadcastTxCommitResult> {
     this.setPrefix()
 
     let account: BaseAccount = (await auth.accountsAddressGet(this.sdk, signer)).data.result
     if (account.account_number === undefined) {
       account = BaseAccount.fromJSON((account as BaseAccountResponse).value)
     }
 
-    return auth.signStdTx(
+    const signedStdTx = auth.signStdTx(
       this.sdk,
       privkey,
       unsignedStdTx,
       account.account_number.toString(),
       account.sequence.toString(),
     )
-  }
 
-  async broadcast(signedStdTx: StdTx): Promise<BroadcastTxCommitResult> {
     return (await auth.txsPost(this.sdk, signedStdTx, 'sync')).data
   }
 
-  async signAndBroadcast(unsignedStdTx: StdTx, privkey: PrivKey, signer: AccAddress): Promise<BroadcastTxCommitResult> {
-    const signedTx = await this.sign(unsignedStdTx, privkey, signer)
-    return await this.broadcast(signedTx)
+  async transferSigned({
+    privkey,
+    from,
+    from_account_number = '0',
+    from_sequence = '0',
+    to,
+    amount,
+    asset,
+    memo = '',
+    fee = {
+      amount: [],
+      gas: '200000',
+    },
+  }: TransferOfflineParams): Promise<StdTx> {
+    this.setPrefix()
+
+    const msg: Msg = [
+      MsgSend.fromJSON({
+        from_address: from,
+        to_address: to,
+        amount: [
+          {
+            amount: amount.toString(),
+            denom: asset,
+          },
+        ],
+      }),
+    ]
+    const signatures: StdTxSignature[] = []
+
+    const unsignedStdTx = StdTx.fromJSON({
+      msg,
+      fee,
+      signatures,
+      memo,
+    })
+
+    return await this.signedOffline(unsignedStdTx, privkey, from_account_number, from_sequence)
+  }
+
+  async signedOffline(
```

Comment:
Is `signedOffline` really needed? Or can we just move line `295` to replace line `284` with it? If so, let's remove `signedOffline` afterwards